# This message holds the data of an object detected by vision
geometry_msgs/Pose object_pose
string id
bool in_view
